nservicebus/upgrades/acs-host-6to7.md

```diff
@@ -8,6 +8,42 @@ related:
 - nservicebus/upgrades/5to6
 ---
 
+## Namespace changes
+
+All types specified in this document are now in the `NServiceBus` namespace.
+
+## IConfigureThisEndpoint changes
+
+`IConfigureThisEndpoint.Customize` is passed an instance of `EndpointConfiguration` instead of `BusConfiguration`.
+
+snippet: AzureServiceBusTransportWithAzureHost
+
+## AsA_Host role changes into IConfigureThisHost
+
+The specifiers `IConfigureThisEndpoint` and `AsA_Host` are now merged into `IConfigureThisHost`. 
+
+snippet: AsAHost
+
+### Removal of DynamicHostControllerConfig
+
+The `DynamicHostControllerConfig`configuration section has been removed, instead the `IConfigureThisHost.Customize` implementation requires to return an instance of `HostSettings` which contains all the configuration values. 
+
+`RoleEnvironment.GetConfigurationSettingValue` can be used to read any existing configuration setting from the `.cscfg` file.
+
+## Removal of dependencies
+
+The host role entry point and host process no longer depend on the following components, feel free to remove them.
+
+ * Log4Net
+ * Ionic.zip
+ * Topshelf
+ * CommonServiceLocator
+
+## Deprecation of profiles
+
+The infrastructure backing profiles has been removed from the host and therefore the `Development` and `Production` profile are no longer available. Note that the effect of of these profiles already disappeared quite a while ago when visual studio introduced native logging. Refer to the [logging documentation](/nservicebus/hosting/cloud-services-host/logging.md) to learn how to set this up.
```

Comment:
> `Development` and `Production` profile

profile**s**

>  of of these profiles

~~of~~ of these profiles

> visual studio

Visual Studio

> Note that the effect of of these profiles already disappeared quite a while ago when visual studio introduced native logging.

Note that those profiles don't have to be used since Visual Studio introduced native logging. Or maybe: Note that using the Visual Studio native logging has the same effect as using the profiles. Or: Note that Visual Studio native logging has replaced the profiles.